packages/frontend/src/features/scheduler/components/SchedulerFilters.tsx

```diff
@@ -399,6 +419,26 @@ const SchedulerFilters: FC<SchedulerFiltersProps> = ({
                             />
                         );
                     })}
+                    {savedFiltersNotInDashboard.length > 0 && (
+                        <Text fz="sm" c="dimmed" fw={500} pt="sm">
+                            The following filters are applied to this scheduled
+                            delivery but no longer exist in the dashboard
+                        </Text>
+                    )}
+                    {savedFiltersNotInDashboard?.map((filter) => {
+                        return (
+                            <Box key={filter.id} pl="md">
+                                <FilterItem
+                                    key={filter.id}
+                                    dashboardFilter={filter}
+                                    schedulerFilter={filter}
+                                    onChange={handleUpdateSchedulerFilter}
+                                    onRevert={() => revertFilter(filter.id)}
+                                    hasChanged={false}
+                                />
```

Comment:
wish there was a way to remove this scheduled delivery filter - I can't remove it from the scheduled delivery after this case